src/routes.py

```diff
@@ -12,15 +15,59 @@ def home():
 	return render_template('index.html', endpoint=endpoint, client_id=client_id, scope=scope,
 		response_type=response_type, redirect_uri=redirect_uri)
 
+def decode_access_token(token):
```

Comment:
route.py is ONLY for routes, move all of these functions to a seperate python file and import them. Don't mix the routes with other functions.